Makefile

```
@@ -1,6 +1,6 @@
-.PHONY: all omnilock anyone-can-pay xudt spore ckb-js-vm nostr-lock ckb-debugger apply-debugger-patches clean-debugger-patches
+.PHONY: all omnilock anyone-can-pay xudt spore ckb-js-vm nostr-lock
 
```

checkmake (0.3.2)

[warning] Required target "clean" is missing from the Makefile.

(minphony)